### PR TITLE
Add .travis.yml for GitHub pages deployment.

### PR DESCRIPTION
**Pull request by davidtwco**
This *should* work, but I've not been able to test on my fork due to a Travis error ("GitHub Pages branch not included via configuration").

cc @spastorino @pietroalbini

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+env:
+  RUSTINFRA_DEPLOY_DIR: public
+import:
+  - rust-lang/simpleinfra/travis-configs/static-websites.yml
+install:
+  - go get github.com/gohugoio/hugo
+before_script:
+  - hugo version
+script:
+  - hugo
```
Comment (spastorino) [inline .travis.yml]: Wouldn't this be `hugo --minify --theme book`?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+language: go
+env:
+  RUSTINFRA_DEPLOY_DIR: public
+import:
+  - rust-lang/simpleinfra/travis-configs/static-websites.yml
+install:
+  - go get github.com/gohugoio/hugo
```
Comment (spastorino) [inline .travis.yml]: I've read somewhere that this is not the recommended way of installing hugo.
Read about doing ...
```
  - curl -LO https://github.com/gohugoio/hugo/releases/download/v0.55.4/hugo_0.55.4_Linux-64bit.deb
  - sudo dpkg -i hugo_0.55.4_Linux-64bit.deb
```
But I'm not sure, I'd say let's investigate that a bit?.

File: .travis.yml (hunk 1 above)
Comment (davidtwco) [reply .travis.yml]: I've added `--minify`. `--theme book` will happen from the configuration file.

File: .travis.yml (hunk 2 above)
Comment (davidtwco) [reply .travis.yml]: I've changed this to use a pinned version.

**Pull request merged**